docs/data/rpc/admin-guide.mdx

```diff
@@ -555,7 +620,7 @@ This test will compile, deploy, and invoke example contracts to the network to w
 
 ```bash
 docker run --rm -t --name e2e_test \
-docker.io/stellar/system-test:soroban-preview11 \
+docker.io/stellar/system-test:soroban-v20 \
```

Comment:
these docker images seem pretty old... might wanna ask around to see if this docker container is used or moved or whatever.

[the repo](https://github.com/stellar/system-test) seems like it's pretty actively maintained, so i would _think_ there are up-to-date container images around somewhere... 🤷🏻‍♂️

Reply:
https://hub.docker.com/r/stellar/system-test/tags
https://github.com/stellar/system-test

While the git repo has commits as recent as last week, the dockerhub latest image is:
[soroban-v20](https://hub.docker.com/layers/stellar/system-test/soroban-v20/images/sha256-aface6b6e4ac8c9cffd9645e3f1a10563cd96e6a61b5d7399ab48888fb2b851a)
Last pushed a year ago by


Any recommendations here?